### ISSUE TITLE
"Reported from in-game: Duplication Glitch" - submitted via. helper utility

### ISSUE THREAD

**Issue by hamstar0** [hamstar0/tml-extensibleinventory-mod#44]
tModLoader version: 0.11.8
 
| Mods:  | - | - |
| :--- | :--- | :--- |
| `Boss Checklist 1.1.4.1` | `Calamity Mod (No Calamity Music) 1.4.5.7` | `Calamity Mod Music 1.4.5.3` |
| `Calamity Mod Extra Music 1.0.7.5` | `Extensible Inventory 1.5.2.1` | `Mod Helpers 5.9.6` |
| `Recipe Browser 0.8.7` | `Thorium Mod 1.6.3.13` | `Which Mod Is This From? (WMITF) 2.6.2` |
 
| Name | Male | Has Demon Heart | Difficulty mode | Life | Mana | Defense |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| Van | False | False | 0 | 132 of 132 (120) | 100 of 100 | 27 |
 
**Player Van's (0) equipment:**
 
| Head | Body | Legs | Accessory 1 | Accessory 2 | Accessory 3 | Accessory 4 | Accessory 5 | Vanity 1 | Vanity 2 |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| Silberhelm | Silberkettenhemd | Silberbeinsch�tzer | Family Heirloom | Silent Wolke in einer Flasche | Gladiator's Locket (Gepanzert) | Heart of Stone (Gezackt) | Silver Bulwark | Schutzbrille | Wiederbelebungsband (Voreilig) |
 
Is day: False, Time of day/night: 8570, Elapsed half days: 13
 
World name: dis tim it sewious, world size: Small
 
World progress: none
 
Items on ground: 88, Npcs active: 12
 
Player count: 1 (single-player)
 
Autopause: False
 
Autosave: True
 
Lighting mode: 0
 
Lighting threads: 1
 
Frame skip mode: 2
 
Is screen maximized: False
 
Screen resolution: 1920 1080
 
Quality style: 1
 
Background on: True
 
Child safety: False
 
Game zoom: 1
 
UI zoom: 1
 
FrameworkVersion.Framework: NetFramework
 
FrameworkVersion.Version: 4.8
 
 
 
Recent error logs:
```
...
[16:21:06] [9/INFO] [StatusText]: Weltenspeichern wird best�tigt
[16:31:06] [16/INFO] [StatusText]: Weltdaten werden gesichert
[16:31:06] [16/INFO] [StatusText]: Weltenspeichern wird best�tigt
[16:41:06] [21/INFO] [StatusText]: Weltdaten werden gesichert
[16:41:07] [21/INFO] [StatusText]: Weltenspeichern wird best�tigt
[16:41:57] [1/DEBUG] [Terraria]: Device Reset, Profile: HiDef, Width: 1920, Height: 1080, Fullscreen: True, Display: \\.\DISPLAY1
[16:47:02] [1/DEBUG] [Terraria]: Device Reset, Profile: HiDef, Width: 1920, Height: 1080, Fullscreen: True, Display: \\.\DISPLAY1
[16:51:06] [16/INFO] [StatusText]: Weltdaten werden gesichert
[16:51:06] [16/INFO] [StatusText]: Weltenspeichern wird best�tigt
[16:54:53] [1/DEBUG] [Terraria]: Device Reset, Profile: HiDef, Width: 1920, Height: 1080, Fullscreen: True, Display: \\.\DISPLAY1
[16:56:46] [1/DEBUG] [Terraria]: Device Reset, Profile: HiDef, Width: 1920, Height: 1080, Fullscreen: True, Display: \\.\DISPLAY1
[17:01:06] [8/INFO] [StatusText]: Weltdaten werden gesichert
[17:01:07] [8/INFO] [StatusText]: Weltenspeichern wird best�tigt
[17:11:06] [8/INFO] [StatusText]: Weltdaten werden gesichert
[17:11:07] [8/INFO] [StatusText]: Weltenspeichern wird best�tigt
```
 
Basically if I have ore in a page that isnt open I can make infinite bars
